src/ecs_deplojo/cli.py

```diff
@@ -12,6 +12,11 @@
 from ecs_deplojo.logger import logger
 from ecs_deplojo.task_definitions import generate_task_definitions
 
+try:
```

Comment:
Do we need this import as it wasn't there before?

Reply:
please also update the python version matrix in tox.ini and the github workflow

everything before 3.9 is EOL and 3.9 is EOL later this year. So:
3.9 (for now), 3.10, 3.11, 3.12 and perhaps even 3.13 would suffice